test/queries_generator/generate_aws_cmds.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#include <fstream>
+
+using namespace std;
+
+int main()
+{
+    fstream query_file, cmd_file;
+    query_file.open("queries.txt", ios::in);
+    cmd_file.open("aws_cmds.sh", ios::out);
+    cmd_file << "#!/bin/sh\nset -x\nset -e\n\n";
```

Comment:
another option is to use popen( ... ) 
https://man7.org/linux/man-pages/man3/popen.3.html
and get rid of the shell